### ISSUE TITLE
Auto-login/Co-ords

### ISSUE THREAD

**Issue by CTCal** [AnthonyOSX/GO-Bot#183]
I'd really like an option for relogging or not if it gets DC'd. Either way that function should probably have a short random timer so it doesn't insta-login every time. I'd mostly want to relog manually after a while, and maybe change locations anyway when it happens. 

Either way instantly relogging and starting at the old coords is just asking to get banned (and that seems to be what it's doing now unless I'm missing something?)

Allowing people to disable automatic relog would be a temporary fix and imo a good feature to have anyway.

Reading through stuff, but haven't coded in ages so I need a lot more time before potentially doing anything myself. Thanks for the work so far!

**Comment by DuIslingr** [comment]
Yes this would be useful for those who don't realize that it doesn't already do this. Would help minimize soft bans from using the bot. It would also be helpful to log the current coordinates the bot is at to reduce soft bans as well.

**Comment by agustinmonti** [comment]
The GO.Bot.v0-0-5 folder has a Coords.txt file that's constantly updated, probably for the relogs.

**Comment by CTCal** [comment]
Yeah, you are correct. Earlier today it restarted 3-4 times over a few minutes and seemed to jump back to the origin in the config, but seems that wasn't the case (it just didn't move a whole lot). The coords do change on the auto relog, still, I'd like to either have an option to stop it or at least have an auto relog on a random timer.

**Comment by ocontant** [comment]
Also, sometime it is better for the bot to stay around the starting point and circle around a little.  By example a popular parc in my area has 8 pokestop in less than 50m radius and they are always with a lure and they generate a lot of pokemon and potential to XP fast.  

Having the bot to go far make it less efficient than if he was looping around these concentration of pokestop with lure.   Would it take a great deal of effort to implement this logic?


Thanks for the great work and taking in consideration this suggestion.
